### PR TITLE
[MERGE] Merge branch 1.5

### PR DESCRIPTION
**Pull request by calvinjia**
Contains the netty fix.

### CODE REVIEW

File: docs/_data/table/en/user-configuration.yml [hunk 1]
```diff
@@ -99,11 +99,12 @@ alluxio.user.short.circuit.enabled:
 alluxio.user.ufs.block.open.timeout.ms:
   N/A
 alluxio.user.ufs.block.read.concurrency.max:
-  N/A
+  The maximum concurrent readers for one UFS block on one Block Worker.
```
Comment (apc999) [inline docs/_data/table/en/user-configuration.yml]: actually, these docs are auto-generated now in branch master. 
The correct way should be updating `PropertyKey.java` with 
```
  public static final PropertyKey USER_UFS_BLOCK_OPEN_TIMEOUT_MS =
      new Builder(Name.USER_UFS_BLOCK_OPEN_TIMEOUT_MS)
          .setAlias(new String[]{"alluxio.user.ufs.block.open.timeout.ms"})
          .setDescription("The maximum concurrent readers for one UFS block on one Block Worker.")
          .setDefaultValue(300000)
          .build();
```

**Comment by apc999** [comment]
up to you if you want to address it in this PR or separate PR.

File: docs/_data/table/en/user-configuration.yml (hunk 1 above)
Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: can we remove the file from revisioning if it is easily autogenerated?

Comment (apc999) [inline docs/_data/table/en/user-configuration.yml]: ok I can make a PR on this

Comment (apc999) [inline docs/_data/table/en/user-configuration.yml]: @calvinjia can you also update `PropertyKey.java` and prevent we lose the info of these four properties.

Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: I think there are only 3 properties updated? I've added the descriptions but I didn't regenerate the docs due to the issue you plan to fix.

**Pull request merged**